hosts_example.yml

```diff
@@ -301,6 +301,35 @@ kafka_connect:
 #     syslog:
 #     elastic:
 
+#### Connectors and the Confluent Hub
+#
+## Enabling/Disabling Confluent Hub Install
+# vars:
+#   kafka_connect_confluent_hub_install: false (Defaults to true)
+#
+## Controling where the Confluent Hub is installed from
+# vars:
+#   kafka_connect_confluent_hub_client:
+#     file: "http://client.hub.confluent.io/confluent-hub-client-latest.tar.gz"
+#     is_remote: yes
+#     install_dir: /etc/kafka/confluent-hub 
+# 
+## Installing Connectors From Confluent Hub
+# vars:
+#   kafka_connect_confluent_hub_plugins:
+#   - connector: jcustenborder/kafka-connect-spooldir:2.0.43
```

Comment:
preferably the customer would just list off connectors, they probably dont care which directory they should get downloaded to (right?). maybe use a default for the directory that the connectors get downloaded to that can be updated.

Does this same logic hold for kafka_connect_plugins?

Reply:
Same logic applies to both. Idea is to work in association with the `plugin.path` value. Since it can be a list (Comma Separated String), I don't know explicitly which dir they want. So opted for them supplying it.

Reply:
Of note, you should see another new task to ensure all `plugin.path` dirs exist (Except the default one).

Reply:
yes but what i mean is the list should look like this
```
# vars:
#   kafka_connect_confluent_hub_plugins:
#   - jcustenborder/kafka-connect-spooldir:2.0.43
# . - whatever/whatever:2.0.3
```

Reply:
That would be ideal if `plugin.path` only accepted a single directory. It however accepts a comma separated list of directories to look in. I opted to support more flexibility by allowing the user to define a destination dir to install to. That dir would be one of those directories listed within `plugin.path`.

Reply:
could we just download the connectors directly into plugin.path[0]